analyzers/src/SonarAnalyzer.CSharp/Rules/NotAssignedPrivateMember.cs

```diff
@@ -194,15 +194,15 @@ private static ISet<ISymbol> GetAssignedMemberSymbols(IList<MemberUsage> memberU
 
                 if (PreOrPostfixOpSyntaxKinds.Contains(parentNode.Kind()))
                 {
-                    assignedMembers.Add(memberSymbol);
+                    AddAssignedMember(assignedMembers, memberSymbol);
```

Comment:
Please refactor this method to use `else if` instead of those `continue` statements